.gitattributes

```diff
@@ -0,0 +1 @@
+* text=auto
```

Comment:
github was showing the whole file as a diff because it got converted to windows file endings, this should prevent that in future and i converted the line endings back